test/api/apikeys_test.go

```diff
@@ -344,23 +344,22 @@ func testAPIKeysAcceptance(t *testing.T, at *test.AccountsTester) {
 		{verb: http.MethodPost, endpoint: "/user/reconfirm"},
 	}
 
-	var b []byte
 	for _, tt := range tests {
 		switch tt.verb {
 		case http.MethodGet:
-			r, b, err = at.Get(tt.endpoint, nil)
+			r, err = at.Request(http.MethodGet, tt.endpoint, nil, nil, nil, nil)
```

Comment:
why not just check whether `tt.verb` is allowed and then do `r, err = at.Request(tt.verb, tt.endpoint, nil, nil, nil, nil)`

Reply:
Oh! That's a great idea. If I use a map with the allowed verbs as keys it will be a single `if`. Thanks!

Reply:
well yeah or you do 
```
switch tt.verb {
    case 1:
    case 2:
    case 3:
    case 4:
         // continue
    default:
         // throw error
}
// use tt.verb
```